### PR TITLE
LPS-175795 Tabs for systems objects are displayed but no entities are listed when mapping an element on a Content Page

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPS-175795](https://issues.liferay.com/browse/LPS-175795)

### CODE REVIEW

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Comment by pedro-oliveira446** [comment]
@marcelabc,
I made some changes and I believe that now, the failures are not related.

**Review by seltong — CHANGES_REQUESTED** [review]

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/item/selector/SystemObjectEntryItemSelectorView.java [hunk 1]
```diff
@@ -105,6 +105,21 @@ public String getTitle(Locale locale) {
 		return _objectDefinition.getPluralLabel(locale);
 	}
 
+	@Override
+	public boolean isVisible(
+		InfoItemItemSelectorCriterion itemSelectorCriterion,
+		ThemeDisplay themeDisplay, String itemSelectedEventName) {
+
+		if (itemSelectedEventName.equals(
+				"_com_liferay_layout_content_page_editor_web_internal_" +
```
[the remark this answers is not in the document]
Comment (pedro-oliveira446) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/item/selector/SystemObjectEntryItemSelectorView.java:116]: @seltong,I couldn't do it.
When I use `ContentPageEditorPortletKeys.CONTENT_PAGE_EDITOR_PORTLET` and do the import, it gives a problem when I start the portal. And about the `themeDisplay`, I haven't found any way to use it to solve the bug.

**Comment by seltong** [comment]
Hi @pedro-oliveira446, could you please check if we can do something like [this](https://github.com/pedro-oliveira446/liferay-portal/pull/18)?

If it works, we don't need to create another `isVisible()` method.

**Review by seltong — CHANGES_REQUESTED** [review]
Hi @pedro-oliveira446, could you please check if we can do something like https://github.com/pedro-oliveira446/liferay-portal/pull/18?

If it works, we don't need to create another isVisible() method.

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
@seltong, 
It works fine, thank you.

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by seltong — APPROVED** [review]

**Comment by seltong** [comment]
LGFM :tada:

**Comment by seltong** [comment]
LGTM :tada:

**Comment by pedro-oliveira446** [comment]
ci:reevaluate:1397401_3892

**Comment by carolmariaabb** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**